anago

```diff
@@ -757,6 +763,7 @@ archive_release () {
 
   ((FLAGS_yes)) || common::askyorn -y "Archive this release on GS" || return
 
+  copy_logs_to_workdir
```

Comment:
archive_release should get the logs as well.  Right now they aren't.  The logs should be copied into the build directory before archiving starts.